perl/README.md

```diff
@@ -1 +0,0 @@
-Don't use perl.
```

Comment:
I will make a slight revision. `Don't use Perl for new development. Only maintain existing internal or third-party Perl.`